src/smc-webapp/project_settings.cjsx

```diff
@@ -827,6 +827,9 @@ HideDeletePanel = rclass
             <Row>
                 <Col sm=8>
                     {@delete_message()}
+                    {<Alert bsStyle="info" style={padding:'8px', marginTop:'4px'} >
+                        <b>New -- </b>This will also clear all upgrades from this project. Undeleting the project will not automatically restore them.
```

Comment:
Why does it say `New --`?    Think of the psychology.  Most SMC users are new or don't remember how things worked before anyways, and nowhere else in SMC do we ever say "heh this is new!".    Also, if we did (which we don't) it should only happen once or for a limited time.  There's nothing worse than explicitly saying something is new and seeing it 10 years later in your codebase.  I bet there is stuff like that in Sage.